src/chart/sunburst/SunburstPiece.js

```diff
@@ -203,7 +203,7 @@ SunburstPieceProto._updateLabel = function (seriesModel, visualColor, state) {
 
     var text = zrUtil.retrieve(
         seriesModel.getFormattedLabel(
-            this.node.dataIndex, 'normal', null, null, 'label'
+            this.node.dataIndex, status, null, null, 'label'
```

Comment:
Where is this `status`? Should it be `state`?

Reply:
Modify clerical error